samples/fundamentals/security/prevent-mixed-content/active-mixed-content.html

```diff
@@ -14,16 +14,16 @@
     <title>Active mixed content example</title>
     
 <!-- An insecure script file loaded on a secure page -->
-    <script src="http://googlesamples.github.io/web-fundamentals/samples/discovery-and-distribution/avoid-mixed-content/simple-example.js"></script>
+    <script src="http://googlesamples.github.io/web-fundamentals/samples/fundamentals/security/prevent-mixed-content/simple-example.js"></script>
```

Comment:
Just realized how old this patch is :/

I can take over if you don't want to bother.